http.go

```diff
@@ -590,3 +619,41 @@ func homeDir() string {
 	}
 	return os.Getenv("HOME")
 }
+
+type loggingTransport struct {
+	transport http.RoundTripper
+	printf    func(f string, a ...interface{})
+}
+
+func (t loggingTransport) RoundTrip(req *http.Request) (*http.Response, error) {
```

Comment:
It might be interesting to format this the way curl -v does.

Reply:
done (ish)